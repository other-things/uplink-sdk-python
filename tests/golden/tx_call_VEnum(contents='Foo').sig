r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 33073731502371413867178951950579459438965189284188600692285471464030988357095
