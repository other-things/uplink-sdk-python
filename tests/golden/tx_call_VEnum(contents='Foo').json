{
    "header": {
        "contents": {
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "args": [
                    {
                        "contents": "Foo",
                        "tag": "VEnum"
                    }
                ],
                "method": "get"
            },
            "tag": "Call"
        },
        "tag": "TxContract"
    },
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE0zMzA3MzczMTUwMjM3MTQxMzg2NzE3ODk1MTk1MDU3OTQ1OTQzODk2NTE4OTI4NDE4ODYwMDY5MjI4NTQ3MTQ2NDAzMDk4ODM1NzA5NQ=="
}